weave/trace/util.py

```diff
@@ -119,6 +119,20 @@ def is_notebook() -> bool:
     return True
 
 
+def _get_func_name_from_maybe_ref(s: str) -> str:
+    """This is a hack and should not be needed. We should be able to auto-resolve this for the user.
+    Keeping this here for now, but it should be removed in the future once we have a better solution.
+    """
+    # it's not a ref, just return same thing
```

Comment:
i would recommend using the parser in `refs.py` rather than doing this prefix matching.

Reply:
much better, thanks!